core/src/main/java/jenkins/fingerprints/FileFingerprintStorage.java

```diff
@@ -124,9 +124,15 @@ public FileFingerprintStorage () {}
     /**
      * Saves the given Fingerprint in local XML-based database.
      */
```

Comment:
```suggestion
     *
```
Seems to be the issue as it ends the comment too early

Reply:
Thanks!